mirrors.yaml

```diff
@@ -27,32 +27,115 @@
     - 22.547, 114.085947
     - 37.514951, 105.189568
     - 31.231706, 121.472644
-- provider: 北京外国语大学
-  url: https://mirrors.bfsu.edu.cn/archlinuxcn/
-  location: 北京
-  geolocs:
-    - Beijing, China
-  frequency: 6h
-  protocols:
-    - ipv4
-    - ipv6
-    - http
-    - https
-  geocoords:
-    - 39.5715, 116.1815
-- provider: 北京大学
-  url: https://mirrors.pku.edu.cn/archlinuxcn/
-  location: 北京
+- provider: 阿里云
```

Comment:
1. I have stated in the commit message that I prioritized the CDN mirrors, but maybe it would be better to sort the CDN mirrors at the end (OpenTUNA synchronization frequency is not high).
2. I have added the synchronization frequency data for each mirror, maybe we can consider sorting them by synchronization frequency?

Reply:
> maybe we can consider sorting them by synchronization frequency?

No. We'd like to avoid TUNA and USTC due to their high load.

Reply:
1. USTC mirror is relatively in the front because its province is Anhui, and its pinyin starts with A.
2. I chose to sort by region because it is convenient to copy the same data from the same region when adding new mirrors later, and this also meets the general mirror sorting requirements.

Reply:
> No. We'd like to avoid TUNA and USTC due to their high load.

The following are the possible solutions I have come up with:

11. Add comments to indicate the status and frequency of USTC and TUNA mirrors
2. Modify update-script to include comments in mirrorlist file
3. Rearrange the order of mirrors by putting mainland China mirrors before CDN mirrors